django/middleware/locale.py

```diff
@@ -35,8 +35,12 @@ def process_response(self, request, response):
         urlconf = getattr(request, 'urlconf', settings.ROOT_URLCONF)
         i18n_patterns_used, prefixed_default_language = is_language_prefix_patterns_used(urlconf)
 
-        if (response.status_code == 404 and not language_from_path and i18n_patterns_used and
+        if (response.status_code == 404 and
+                not language_from_path and
+                i18n_patterns_used and
                 (prefixed_default_language or language != settings.LANGUAGE_CODE)):
```

Comment:
Django's settings for flake8 does not allow breaking before binary operators (flake8 W503). (https://github.com/django/django/blob/master/setup.cfg#L7)
I will update the PR if this warning is ignored.